docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -9,6 +9,35 @@ Release Notes
 Features
 ========
 
+New in Version 4.2.1
+--------------------
+The major changes for this point release were to fix several problems with using
+the built in editor to create new models and to bring the NXcanSAS reader into
+compliance with the final published specification.  The orignal reader was
+based on a draft version of the specification.  As an early adopter,
+interpretation and implementation of the spec was iterated with all producers
+of NXcanSAS reduced data known to the SasView team in order to ensure
+compatibility and verify the implementation.  A few other enhancements and bug
+fixed were also introduced such as cleaning up the resolution section of the
+fitting page GUI, increasing the max size range allowed in the corfunc
+analysis, and adding the incomplete gamma function to the python library. 
+
+Resolved Issues
+^^^^^^^^^^^^^^^
+* Fixes #  976:	CanSas HDF reader will not read all valid CanSas HDF (NXcanSAS) files
+* Fixes # 1074:	Add incomplete gamma function to sasmodels
+* Fixes # 1111:	Convert all input Q units to 1/A
+* Fixes # 1129:	NXcanSAS writer not writing all meta data
+* Fixes # 1142:	Plugin framework is broken
+* Fixes # 1183:	Test from creating new model reset all parameters to default in all open FitPages
+* Fixes # 1205:	4.2 set weighting choice seems to be ignored.
+* Fixes # 1206:	Incorrect (and confusing) presentation of dQ from data in instrumental smearing section
+* Fixes # 1212:	Bug in Iqxqy plotting non rectangular / square matrices?
+* Fixes # 1221:	ABS reader does not read in USANS data properly	GitHub
+* Fixes # 1222:	smearing options incorrect on show2D and show1D in fitpage14: Loading a saved project is really really slow
+* Fixes # 1223:	Expand permitted range of transformed data in Corfunc implementation
+
+
```

Comment:
I removed the ':' from magnetic parameter names since the 0.98 sasmodels release.  The saved models should be adjusted automatically on load, but you may want to mention the change in the release notes.  Here's a diff that shows the renaming:
```
+def _rename_magnetic_pars(pars):
+    """
+    Change from M0:par to par_M0, etc.
+    """
+    keys = list(pars.items())
+    for k in keys:
+        if k.startswith('M0:'):
+            pars[k[3:]+'_M0'] = pars.pop(k)
+        elif k.startswith('mtheta:'):
+            pars[k[7:]+'_mtheta'] = pars.pop(k)
+        elif k.startswith('mphi:'):
+            pars[k[5:]+'_mphi'] = pars.pop(k)
+        elif k.startswith('up:'):
+            pars['up_'+k[3:]] = pars.pop(k)
+    return pars
```

I also twiddled the handling of multiple scattering, but this isn't available yet through the sasview interface.

Spinodal docs have been updated.